go/frontendbridge/main.go

```diff
@@ -69,15 +71,15 @@ func (w *FrontendBridge) addConnection(auth structs.MessageAuth, socket *websock
 }
 
 func (w *FrontendBridge) getConnection(auth structs.MessageAuth) *Connection {
-	w.connectionsMutex.Lock()
+	w.connectionsMutex.RLock()
 	var result *Connection
 	for _, connection := range w.connections {
 		if reflect.DeepEqual(auth, connection.auth) {
 			result = connection
 			break
 		}
 	}
-	w.connectionsMutex.Unlock()
+	w.connectionsMutex.RUnlock()
```

Comment:
I am searching through the available connections looking for one where the auth matches. This should be fast and I can't think of any better way to do this